components/sidebar/sidebar_service.cc

```diff
@@ -611,6 +537,86 @@ void SidebarService::LoadSidebarItems() {
   }
 }
 
+std::vector<SidebarItem> SidebarService::GetDefaultSidebarItems() const {
+  std::vector<SidebarItem> items;
+  for (const auto& item_type : SidebarService::kDefaultBuiltInItemTypes) {
+    if (auto item = GetBuiltInItemForType(item_type);
+        item.built_in_item_type != SidebarItem::BuiltInItemType::kNone) {
+      items.push_back(item);
```

Comment:
can we add move/copy defaults to `SidebarItem` and do `std::move(item)` here?

Reply:
In this PR, this code was just moved. How about handling this in a separated PR?
I'll f/u this right after this PR is merged.